<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STD_MaterialWidget</class>
 <widget class="QWidget" name="STD_MaterialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="gbMaterial">
      <property name="title">
       <string>Material List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="RB_ToolButtonBar" name="tbbMaterial" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="RB_TableView" name="tvMaterial"/>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Details</string>
       </attribute>
       <widget class="QLabel" name="lblName">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leName">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblNumber">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leNumber">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblType">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbType">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblClass">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Class</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbClass">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cbStructure">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>150</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblStructure">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Structure</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblComposition">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Composition</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leComposition">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>190</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblDocument">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Document</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leDocument">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblIssue">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>50</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Issue</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leIssue">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblElasModulTable">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>80</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Elasticity Modulus Table</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbElasModulTable">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>80</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblThermExpTable">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>120</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Thermal Expansion Table</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbThermExpTable">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>110</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblCreepTable">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Creep Table</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbCreepTable">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>150</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lblExtPress">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>190</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>External Pressure Table</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbExtPressTable">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>180</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>EN Stress Values</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="gbRp02">
          <property name="title">
           <string>0.2% Proof/Yield Strength</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="RB_ToolButtonBar" name="tbbRp02" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="RB_TableView" name="tvRp02"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="gbRp10">
          <property name="title">
           <string>1.0% Proof/Yield Strength</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="RB_ToolButtonBar" name="tbbRp10" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="RB_TableView" name="tvRp10"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="gbRmMin">
          <property name="title">
           <string>Minimum Tensile Strength</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="RB_ToolButtonBar" name="tbbRmMin" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="RB_TableView" name="tvRmMin"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RB_ToolButtonBar</class>
   <extends>QWidget</extends>
   <header>rb_toolbuttonbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RB_TableView</class>
   <extends>QTableView</extends>
   <header>rb_tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
